lib/widgets/message_list.dart

```diff
@@ -1462,10 +1533,11 @@ class MessageWithPossibleSender extends StatelessWidget {
               Expanded(child: Column(
                 crossAxisAlignment: CrossAxisAlignment.stretch,
                 children: [
-                  MessageContent(message: message, content: item.content),
+                  content,
                   if ((message.reactions?.total ?? 0) > 0)
                     ReactionChipsList(messageId: message.id, reactions: message.reactions!),
-                  if (editStateText != null)
+                  if (editMessageErrorStatusRow != null) editMessageErrorStatusRow
```

Comment:
I think this would go well as its own widget:
```suggestion
                  if (editMessageErrorStatus != null)
                    _EditMessageStatusRow(messageId: message.id),
```

That'd help keep this build method mostly clear of details that are specific to a message currently being edited.